scripts/ngi_aggregate_statistics

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
+
+""" 
+This is a script to create NGI aggregate report.
```

Comment:
I mostly put it in a separate script for two reasons. Firstly I was kind of unsure of where it would be appropriate to put things. Secondly I thought that this was information that you might want to generate independently of the creating the sample reports etc - e.g. to do some quick qc of a project before it's actually finished.

What do you think of moving the the code into `common/project_summary.py` and then keeping a separate utility script to only generate the csv? That way one could quickly generate a `csv` file without having to wait for plots (see my comment about the plotting above).